ghost/core/core/frontend/helpers/content.js

```diff
@@ -19,7 +19,8 @@ function restrictedCta(options) {
     options = options || {};
     options.data = options.data || {};
     _.merge(this, {
-        accentColor: (options.data.site && options.data.site.accent_color)
+        accentColor: (options.data.site && options.data.site.accent_color),
+        resource: options.data.root?.page ? 'page' : 'post'
```

Comment:
@ErisDS Probably using `#is` helper in the CTA to render `page` vs `post` conditionally instead of this pattern is better here ? Was originally inclined towards this as it gives easy way for custom content helpers to render `page` vs `post`, but realized later the use `#is` here is more consistent without adding any new variables. Wdyt ?

Reply:
Updated. ✅